### PR TITLE
feat(apps/interface): add unit test for RedIndicator

### PR DESCRIPTION
**Pull request by asaadam**
add unit test for RedIndicator

Fix RIS-586

### CODE REVIEW

**Review by math-marcellino — COMMENTED** [review]
I'll leave the test review to @pyk coz I'm also new to testing .-.

But I have a bit of comment. Next time try to copy the branch name directly from linear.

image

That way, when you created a PR or it is approved & merged, it will be automatically updated in linear

File: apps/interface/tests/components/RedIndicator.spec.tsx [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { RedIndicatorIcon } from "@/components/Icons/RedIndicator";
+import { render, screen } from "@testing-library/react";
```
Comment (pyk) [inline apps/interface/tests/components/RedIndicator.spec.tsx:2]: Import the 3rd party libs first then our internal component.

Otherwise all looks good

**Review by pyk — CHANGES_REQUESTED** [review]

**Comment by pyk** [comment]
> But I have a bit of comment. Next time try to copy the branch name directly from linear.

Yea we can do this, or include this text on the pull request decription:

```
Fix RIS-NUMBER
```

**Review by math-marcellino — APPROVED** [review]

**Review by pyk — APPROVED** [review]
LGTM. Thanks @asaadam 🔥

**Pull request merged**